packages/core/src/controllers/OptionsController.ts

```diff
@@ -135,6 +135,11 @@ export interface OptionsControllerStatePublic {
    * @default false
    */
   allowUnsupportedChain?: boolean
+  /**
+   * Enable manual control of WalletConnect connections.
+   * @default false
+   */
+  useInjectedUniversalProvider?: boolean
```

Comment:
If it's internal only, we might need to move it from this object since user's will see it and might try to use it

Reply:
true, nice catch